### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by raghav6202**
Please review the cart page component

image

### CODE REVIEW

File: card.css [hunk 1]
```diff
@@ -56,6 +56,171 @@
...
+  
+  .card-heading{
+    font-size: 1.5rem;
+    font-weight: 500;
+    padding: .5rem 1rem 0;
+  }
+  
+  .card-text{
+    font-size: .9rem;
+    color: #444444;
```
Comment (NeerajVodala) [inline card.css:129]: try to use variables for color

File: card.css [hunk 2]
```diff
@@ -56,6 +56,171 @@
...
+    border-radius: 0.5rem;
+}
+
+.horizontal-card-info
+{
+    display: flex;
+    flex-flow: column wrap;
+    gap: 0.5rem;
+    padding: 0.5rem;
+    margin: 00.25rem;
```
Comment (NeerajVodala) [inline card.css:73]: looks like a typo, please correct.

File: card.css [hunk 3]
```diff
@@ -56,6 +56,171 @@
...
+.vertical-card{
+    height: max-content;
+    width:300px;
+    background-color: white;
+    border-radius: 2px;
+  
+  }
+  
+  
+  .card-heading{
```
Comment (NeerajVodala) [inline card.css:121]: inconsistent spacing(use an extension to avoid them)

**Review by NeerajVodala — COMMENTED** [review]
- Try to use rems over px wherever possible
- inconsistent spacing in the code(can prevent that using any extension)
- can use variables for colors, box-shadow, font weight.

Rest everything looks good to me.
All the best.

**Review by bharati-21 — APPROVED** [review]
Only one suggestion - to use a formatter and ensure consistent spacing in the code.

Looks good to be merged.

**Comment by prath-dev** [comment]
Nice Color combination Elegant UI

**Pull request merged**